runtime/codert_vm/decomp.cpp

```diff
@@ -1348,31 +1348,43 @@ freeDecompilationRecord(J9VMThread * decompileThread, J9JITDecompilationInfo * i
 void  
 jitDataBreakpointAdded(J9VMThread * currentThread)
 {
+	J9JavaVM *vm = currentThread->javaVM;
+	J9JITConfig *jitConfig = vm->jitConfig;
+
 	/* We have exclusive */
 
 	Trc_Decomp_jitDataBreakpointAdded_Entry(currentThread);
 
-	++(currentThread->javaVM->jitConfig->dataBreakpointCount);
+	++(jitConfig->dataBreakpointCount);
 
-	/* Remove all breakpoints before resetting the methods */
+	bool inlineWatches = J9_ARE_ANY_BITS_SET(vm->extendedRuntimeFlags, J9_EXTENDED_RUNTIME_JIT_INLINE_WATCHES);
```

Comment:
This probably deserves a comment to mention that its a one-way switch.  Once inlineWatches is enabled and data breakpoints have been set, this can;t be disabled (at least not currently)

Reply:
This actually isn't an appropriate place for that comment - I'll add a different one, and put the comment you want in the data BP removed code.